recipes/paml/4.9/baseml.ctl

```diff
@@ -0,0 +1,34 @@
+      seqfile = brown.nuc 
+     treefile = brown.trees
+
+      outfile = mlb       * main result file
+        noisy = 2   * 0,1,2,3: how much rubbish on the screen
+      verbose = 0   * 1: detailed output, 0: concise output

```

Comment:
@gregvonkuster in general we do not need to test tools with real inputs. This you can do on Galaxy. A simple `--version` or similar is enough.